src/electron/main.ts

```diff
@@ -31,6 +31,7 @@ async function createWindow() {
     windowOptions.webPreferences = {
         nodeIntegration: true,
         webSecurity: false,
+        enableRemoteModule: true
```

Comment:
Required after the update to Electron 10, since the `remote` module is disabled by default.